### PR TITLE
SALTO-5991: Delete config upgrade from Okta and document new elem ID config

### PR DESCRIPTION
**Pull request by shir-reifenberg**
The upgrade completed so I think we can remove the upgrade code

---

_Additional context for reviewer_
Also added to the config doc the `elemID` documentation
please review text & structure as I'm gonna copy it to service placeholder adapter

---
_Release Notes_: 
None 
TODO - update

---
_User Notifications_: 
None

### CODE REVIEW

File: packages/okta-adapter/config_doc.md [hunk 1]
```diff
@@ -73,13 +73,14 @@ okta {
...
-| convertUsersIds                 | true                   | When enabled, user IDs will be replaced with user login names       |
-| includeGroupMemberships         | false                  | Include group assignments                                           |
-| includeProfileMappingProperties | true                   | Include profile mapping properties for profile mapping instances    |
+| Name                                | Default when undefined | Description                                                         |
+| ----------------------------------- | ---------------------- | ------------------------------------------------------------------- |
+| [include](#fetch-entry-options)     | [{ type = ".*" }]      | List of entries to determine what instances to include in the fetch |
+| [exclude](#fetch-entry-options)     | []                     | List of entries to determine what instances to exclude in the fetch |
+| [elemID](#element-id-customization) | {}                     | Allows customizing element IDs for specific types                   |
```
Comment (shir-reifenberg) [inline packages/okta-adapter/config_doc.md:80]: {} fix to empty mappings

File: packages/okta-adapter/config_doc.md [hunk 2]
```diff
@@ -96,8 +97,25 @@ okta {
...
 
+## Element ID parts definitions
+
+| Name        | Default when undefined | Description                                                                         |
+| ----------- | ---------------------- | ----------------------------------------------------------------------------------- |
+| fieldName   | system's default       | The instance's field name value will be used to create the element ID               |
+| isReference | false                  | Set to true if the fieldName is also a Salto reference                              |
+| mapping     | undefined              | Allows applying a specific function to fieldName. Options: 'uppercase', 'lowercase' |
+
+## Element ID customization
```
Comment (shir-reifenberg) [inline packages/okta-adapter/config_doc.md:108]: switch order

**Review by netama — COMMENTED** [review]
I don't know if it's good to remove the system config doc now, we should convert it to a different format but should still have an easy documentation of what the "system" definitions are (can discuss IRL)

**Comment by shir-reifenberg** [comment]
> I don't know if it's good to remove the system config doc now, we should convert it to a different format but should still have an easy documentation of what the "system" definitions are (can discuss IRL)

I can leave the file as is, but this is no longer maintained now that we have the new fetch definitions + some of it is not true (like the pages types)

**Pull request merged**